source/more-info/unsupported/network_manager.markdown

```diff
@@ -9,7 +9,7 @@ The Supervisor utilizes the Network Manager on the host to offer network informa
 to add-ons and to give you the option to manage the network interfaces using the UI
 or via the command-line.
 
-This requires the the Network Manager to be installed, active
+This requires the Network Manager to be installed, active
```

Comment:
It feels like that we talking about NetworkManager (the software part) and not the manager for the network which is NetworkManager.

```suggestion
This requires NetworkManager to be installed, active
```